cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1818,6 +1818,10 @@ public static String getZKQuorum(CConfiguration cConf) {
      * Prefix of per-client TMS topic used on the tethering server.
      */
     public static final String TOPIC_PREFIX = "tethering.topic.prefix";
+    /**
+     * TMS topic used on tethering client for received control messages.
+     */
+    public static final String TETHERING_TOPIC = "tethering.topic";
```

Comment:
I believe we decided that a single topic would be fine, so the programs will be triggered based only on when the request comes in (instead of also depending on which peer sent the request)